rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -891,6 +894,16 @@ def _update_participant_summary(
                                 QuestionnaireStatus.SUBMITTED_NO_CONSENT
                             participant_summary.consentForElectronicHealthRecordsAuthored = authored
                             participant_summary.consentForElectronicHealthRecordsTime = questionnaire_response.created
+                    elif code.value.lower() == REMOTE_ID_VERIFIED_ON_CODE:
+                        utc_datetime = datetime(
+                            answer.valueDate.year,
+                            answer.valueDate.month,
+                            answer.valueDate.day,
+                            tzinfo=pytz.timezone('UTC')
+                                            )
```

Comment:
Do we need to do this? I think the value will be stored as UTC time on insertion. I haven't seen anywhere else where we do this.

Reply:
I am not seeing anywhere else we do this, however when I leave just the answer.valueDate I am receiving this error:
`sqlalchemy.exc.StatementError: (builtins.AttributeError) 'datetime.date' object has no attribute 'tzinfo'`

Reply:
Ah, ok. Sorry I hadn't really thought through that we're storing just the date here. I think the verified_on column should just be a Date datatype then (rather than a variation of Datetime). That way we can store just the date value sent and not have to do anything with timezones.